src/components/Table/TableScan.vue

```diff
@@ -87,6 +108,19 @@
     return result.trim();
   }
 
+  const tableActions = [
+    {
+      name: ScanActions.insight,
+      icon: 'fas fa-chart-simple',
+      show: (value: string): boolean => value === ScanStatus.completed
+    },
+    {
+      name: ScanActions.cancel,
+      icon: 'close',
+      show: (value: string): boolean => value === ScanStatus.inProgress
```

Comment:
There may be an edge case here where a Scan may be taking very very long to start, and the user wishes to cancel it. In this case, the status would be Pending, therefore:

```suggestion
      show: (value: string): boolean => value === ScanStatus.inProgress || value === ScanStatus.pending
```

Reply:
Please don't forget about this condition.